animation_nodes/nodes/mesh/set_vertex_weights.py

```diff
@@ -0,0 +1,124 @@
+import bpy
+from bpy.props import *
+from ... events import propertyChanged
+from ... data_structures import VirtualDoubleList
+from ... base_types import AnimationNode, VectorizedSocket
+
+modeItems = [
+    ("ALL", "All", "Set weight of every vertex", "NONE", 0),
+    ("INDEX", "Index", "Set weight of a specific vertex", "NONE", 1)
+]
+
+groupIdentifierTypeItems = [
+    ("INDEX", "Index", "Get vertex group based on the index", "NONE", 0),
+    ("NAME", "Name", "Get vertex group based on the name", "NONE", 1)
+]
+
+class VertexWeights(bpy.types.Node, AnimationNode):
```

Comment:
Rename the class to `SetVertexWeight`, making `weight` singular in the file name and the `bl_` properties along the way.

Reply:
I have renamed it something like this:
```
class SetVertexWeightNode(bpy.types.Node, AnimationNode):
    bl_idname = "an_SetVertexWeightNode"
    bl_label = "Set Vertex Weight"
```
does it fine ?

Reply:
Yes.